app/managers.py

```diff
@@ -0,0 +1,46 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("../cinema.sqlite")
+        self._table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self._table_name} (first_name, last_name) "
+            f"VALUES (?, ?)",
```

Comment:
Using f-strings to construct SQL queries can lead to SQL injection vulnerabilities. It's safer to use parameterized queries, which you are already doing with placeholders `?`. Ensure that the table name is not user-controlled to prevent injection through the table name.